projects/plugins/beta/src/admin/admin.js

```diff
@@ -206,7 +206,7 @@
 	 * @returns {string} The massaged search string.
 	 */
 	function pr_to_header( search ) {
-		return search.replace( '/', ' / ' ).replace( '-', ' ' ).replace( /  +/g, ' ' ).toLowerCase();
+		return search.replace( /\//g, ' / ' ).replace( /-/g, ' ' ).replace( /  +/g, ' ' ).toLowerCase();
```

Comment:
```suggestion
		return search.replace( /\//g, ' / ' ).replace( /-/g, ' ' ).replace( /  +/g, ' ' ).toLowerCase().trim();
```

What do you think about adding a trim here? Sometimes I copy/paste branch names from terminal and pick up extra spacing so it doesn't match.

Other than that, searching for branches with multiple `-` worked for me 👍

Reply:
Yeah, that seems like a good idea. Committed.